scene/gui/tree.cpp

```diff
@@ -5494,22 +5494,30 @@ void Tree::_do_incr_search(const String &p_add) {
 	ensure_cursor_is_visible();
 }
 
-TreeItem *Tree::_find_item_at_pos(TreeItem *p_item, const Point2 &p_pos, int &r_column, int &h, int &section) const {
+TreeItem *Tree::_find_item_at_pos(TreeItem *p_item, const Point2 &p_pos, int &r_column, int &r_height, int &r_section) const {
+	r_column = -1;
+	r_height = 0;
+	r_section = -1;
```

Comment:
```suggestion
	r_section = -100;
```
`-1` is a valid section indicator, `get_drop_section_at_position` returns `-100` as invalid one, these should be consistent.
(and yeah, they could be constants...)